scb-scanprocesses/zap-process/src/main/java/io/securecodebox/scanprocess/zap/constants/ZapProcessVariables.java

```diff
@@ -0,0 +1,5 @@
+package io.securecodebox.scanprocess.zap.constants;
+
+public enum ZapProcessVariables {
+    ZAP_SKIP_SPIDER;
```

Comment:
I think we should probably remove the ZAP_ prefix. 

@J12934 whats your meaning?

Reply:
You are right. I need to keep the attribute names because they should be consistent and need to match the microservice attribute names (this would be another refactoring step). But there is no need to use the prefix for the process variables, which are used in the engine only.